src/main/kotlin/stringcalculator/domain/StringTokenizer.kt

```diff
@@ -0,0 +1,24 @@
+package stringcalculator.domain
+
+class StringTokenizer {
+
+    companion object {
+        private const val COMMA_DELIMITER = ","
+        private const val COLON_DELIMITER = ":"
+        private val DEFAULT_DELIMITERS = arrayOf(COMMA_DELIMITER, COLON_DELIMITER)
+        private const val DEFAULT_SUFFIX_OF_CUSTOM_DELIMITER = "//"
+        private const val DEFAULT_PREFIX_OF_CUSTOM_DELIMITER = "\n"
+
+        fun tokenize(numbersString: String): Tokens {
+            val customDelimiter = Regex("$DEFAULT_SUFFIX_OF_CUSTOM_DELIMITER(.)$DEFAULT_PREFIX_OF_CUSTOM_DELIMITER(.*)").find(numbersString)
+            customDelimiter?.let {
+                val customDelimiter = it.groupValues[1]
```

Comment:
[destructured](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-match-result/destructured.html)를 이용하면 아래와 같이도 사용이 가능해요!!

```suggestion
                val (customDelimiter, matchedNumbersString) = it.destructured
```